Ginger/GingerCoreNETUnitTest/ValueExpressionTest/ValueExpressionTest.cs

```diff
@@ -19,16 +19,94 @@
 using GingerCoreNET.RosLynLib;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Repository;
+using Ginger.Repository;
+using GingerCore;
+using GingerCore.Environments;
+using GingerCore.Variables;
+using GingerTestHelper;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using GingerCore.Actions;
+using Ginger.Run;
+using System.IO;
+using Amdocs.Ginger.CoreNET.Repository;
+using GingerCoreNETUnitTest.RunTestslib;
 
 namespace GingerCoreNETUnitTests.ValueExpressionTest
 {
     [TestClass]
     public class ValueExpressionTest
     {
+        RunSetConfig runset;
```

Comment:
Codacy found an issue: [Remove the field 'runset' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6384597)